feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailViewModel.kt

```diff
@@ -51,8 +53,17 @@ class MovieDetailViewModel @Inject constructor(
       .collect { result ->
         result.fold(
           onSuccess = { data ->
-            _state.update { state -> state.copy(videoDetail = data, isLoading = false) }
+            _state.update { state ->
+              state.copy(
+                videoDetail = data,
+                isLoading = false,
+                hasCollection = data.collectionId != null,
+              )
+            }
             loadRatings()
+            data.collectionId?.let { id ->
+              loadCollection(id.toInt())
+            }
```

Comment:
💅💅💅 to make it cleaner we can check `data.collectionId` nubility  inside `loadCollection` function